labop/execution_engine.py

```diff
@@ -500,7 +583,7 @@ def _make_object_edge(dot, incoming_flow, target, dest_parameter=None):
         else:
             edge_label = f"{value}"
 
-        if hasattr(value, "to_dot") and not is_ref:
+        if False and hasattr(value, "to_dot") and not is_ref:
```

Comment:
What purpose does the `if False` clause serve?  Doesn't this just permanently disable the subsequent code?